bin/test-workflow/policy/app-access.yml

```diff
@@ -50,3 +50,20 @@
       role: !layer /test-app
       privileges: [ read, execute ]
       resources: *secretless-variables
+
+- !policy
+  id: test-secrets-provider-init-app-db
+  owner: !group secrets_admin

```

Comment:
Not certain about this, but maybe this group should be `test-apps` or something similar, rather than `secrets_admin`? `secrets_admin` seems to point to a Conjur security admin, but this policy is for allowing apps to read secrets, IIUC.

Reply:
Changed to `developer`